test/test_fx_experimental.py

```diff
@@ -550,6 +551,28 @@ def foo(x):
 
         traced = symbolic_trace_with_rewrite(foo)
 
+    def test_fetch(self):
+        class TestModule(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv = torch.nn.Conv2d(3, 3, 2)
+                self.bn = torch.nn.BatchNorm2d(3)
+
+            def forward(self, a):
+                a = self.conv(a)
+                a += a
+                return self.bn(a)
+
+        mod = TestModule()
+        traced = symbolic_trace(mod)
+        lift_lowering_attrs_to_nodes(traced)
+        for node in traced.graph.nodes:
+            if node.op == "call_module":
+                assert hasattr(node, "attrs_for_lowering")
+                _, para_list = module_fetch_book[locate(node.attrs_for_lowering["name"])]
```

Comment:
As part of good testing practice, you shouldn't use the code you're trying to test as a reference in the test. I suggest just having a list of expected attributes for the Conv and BatchNorm node and assert that they are present manually.